### DISCUSSION TITLE
Repository is inactive but development has resumed on another

### DISCUSSION THREAD

**Discussion by DereC4** [jely2002/youtube-dl-gui#496] [General]
Looks like https://github.com/StefanLobbenmeier/youtube-dl-gui has a gui that's constantly updated; on this repo we haven't received any activity on issues and whatnot for a while

**Comment by leoplaw** [comment]
Thanks for this!

**Comment by jely2002** [comment]
Thanks for pointing users to that fork! Closing this now as this repo has resumed development with v3.
